docs/user/explanations/epinio-push-detailed.svg

```diff
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Please click the three dots at the top right of this box and then "View file" to review the changes here.